include/DecayTree.h

```diff
@@ -76,6 +76,9 @@ class DecayTree
     const DaughterDecayTreeMap daughterDecayTrees() const
     { return DaughterDecayTrees_; }
 
+    /// \return DaughterDecayTrees as vector
+    const DecayTreeVector daughterDecayTreeVector() const;
```

Comment:
does not need to be a member function since we have access to `DaughterDecayTrees_`. name suggestion: `as_vector(...)`